pom.xml

```diff
@@ -53,7 +53,7 @@
 		<htmlformentryVersion>4.1.0</htmlformentryVersion>
 		<htmlformentryuiVersion>2.0.0</htmlformentryuiVersion>
 		<coreappsVersion>1.33.0</coreappsVersion>
-		<webservices.restVersion>2.32.0-SNAPSHOT</webservices.restVersion>
+		<webservices.restVersion>2.31.0</webservices.restVersion>
```

Comment:
hi @dkayiwa  how does this pr look to you

Reply:
> is there any reason why you did not just release version 2.32.0 of the module?

@dkayiwa  its because i had already released version 2.31.0!

Reply:
@HerbertYiga it has been a while since that was done. And a number of strategic commits have been done to that module and a few others. My suggestion would be going through them again and see if even more modules should be released and included in the upcoming reference application.

Reply:
> it has been a while since that was done. And a number of strategic commits have been done to that module and a few others. My suggestion would be going through them again and see if even more modules should be released and included in the upcoming reference application.

well noted